Extras/MinimalStreamTester/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: "3.4"
+services:
+  signalling:
+    build:
+      context: ../../
+      dockerfile: ./SignallingWebServer/Dockerfile
+    image: signalling
+    command: --rest_api
+    networks:
+      - testing
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://signalling:80"]
+        interval: 10s
+        timeout: 5s
+        retries: 3
+        start_period: 30s
+        start_interval: 5s # only supported on docker engine 25+
+
+  streamer:
+    image: mcottontensor/ps-minimal-streamer-linux
```

Comment:
Can we move this image to this dockerhub: https://hub.docker.com/u/pixelstreamingunofficial